src/pack.test.ts

```diff
@@ -2,24 +2,28 @@ import { pack } from './index';
 
 jest.setTimeout(720_000);
 
-test('Pack', async () => {
-    const result = await pack({
-        baseDir: './tests/in/pack-01',
-        outDir: './tests/out/pack-01',
-        modules: [{
-            name: 'a',
-            sourceDir: './a',
-        }, {
-            name: 'b',
-            sourceDir: './b',
-        }, {
-            name: 'c',
-            sourceDir: './c',
-        }],
-    });
+describe('packer', () => {
+    test('pack', async () => {
```

Comment:
Not a blocker for this PR, but since we both use `it()` and `test()` across the tests in this repo, we should try to establish a standard